keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/orca/OrcaExecutionUtils.kt

```diff
@@ -0,0 +1,59 @@
+package com.netflix.spinnaker.keel.orca
+
+import java.time.Duration
+
+class OrcaExecutionUtils {
+  companion object {
```

Comment:
We could just put these functions at package level rather than needing a utils class